### PR TITLE
Server-side code for map markers

### PR DESCRIPTION
**Pull request by dalisc**
Users can add a marker onto a map and input info about that place which appears on an info box.

https://dchan-sps-summer20.appspot.com/

You might wanna skip commit f443adb (Add marker object and servlet). I didn't check if the code could compile before committing it and there were typos! They are fixed and can be compiled in the next commit.

### CODE REVIEW

File: portfolio/src/main/webapp/assets/js/formValidation.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  var lat, lng, link;
+  var latError, lngError, linkError;
+
+  const urlRegex = /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)?/;
+
+  lat = document.forms["mapForm"]["lat"].value;
+  lng = document.forms["mapForm"]["lng"].value;
+  link = document.forms["mapForm"]["link"].value;
+
+  console.log(lat, lng, link)
```
Comment (123cristen) [inline portfolio/src/main/webapp/assets/js/formValidation.js:11]: Consider removing these before submitting :)

File: portfolio/src/main/webapp/assets/js/script.js [hunk 2]
```diff
@@ -16,99 +16,102 @@
...
-document.addEventListener("DOMContentLoaded", function () {
-  addFilmImages();
-  getComments();
+document.addEventListener("DOMContentLoaded", function() {
+    addFilmImages();
+    getComments();
 });
 
+function zoomOut() {
+    console.log("trigger");
```
Comment (123cristen) [inline portfolio/src/main/webapp/assets/js/script.js:25]: Do you need this?

**Review by 123cristen — APPROVED** [review]
Awesome!!

File: portfolio/src/main/webapp/assets/js/script.js (hunk 2 above)
Comment (dalisc) [reply portfolio/src/main/webapp/assets/js/script.js:25]: sorry! these always slip through the cracks soomehow

**Pull request merged**